partials/app-banner.html

```diff
@@ -1,18 +1,20 @@
-<div id="appBanner" class="ncf__app-banner">
-	<div class="ncf__app-banner-inner">
-		<div class="ncf__app-banner-image">
-			<img alt="FT app" src="https://www.ft.com/__origami/service/image/v2/images/raw/https://s3-eu-west-1.amazonaws.com/envoy-b2b-trial/phone.png?source=ip-envoy" />
-		</div>
-		<div class="ncf__app-banner-content">
-			<h2 class="ncf__app-banner-header">Get our free App</h2>
-			Download the <span class="ncf__app-banner-strong">FT app</span> to access articles on the move.
-		</div>
-		<div class="ncf__app-banner-actions">
-			<div class="ncf__app-banner-action ncf__app-banner-action--ios">
-				<a href="https://itunes.apple.com/app/apple-store/id1200842933?pt=246269&ct=onsite-app-promotion&mt=8" target="_blank" role="link"><img src="https://www.ft.com/__assets/creatives/tour/apps/ios-download.svg" alt="Download from the iOS App store" height="35"></a>
+<div class="ncf">
+	<div id="appBanner" class="ncf__app-banner">
```

Comment:
Can you not do this?

```suggestion
	<div id="appBanner" class="ncf ncf__app-banner">
```